src/flows/pipes/MetricSelector/view.tsx

```diff
@@ -35,7 +35,7 @@ const DataSource: FC<PipeProp> = ({Context}) => {
   }
 
   return (
-    <BucketProvider>
+    <BucketProvider scope={scope}>
```

Comment:
does the `scope` refer to notebook, data explorer, and wherever the `BucketProvider` is used?

Reply:
yeah. so data explorer has a fixed scope of `{ org: org.id, region: window.location }`, but for notebooks you can query across regions so each panel needs to have a bucket provider with a potentially different scope